intake/gui/tests/test_gui.py

```diff
@@ -15,5 +15,5 @@ def test_add_cat():
     import intake
     intake.gui.add_cat(os.path.join(here, '..', '..', 'catalog', 'tests',
                                     'catalog1.yml'))
-    assert 'catalog1' in intake.gui.cat_list.value
+    assert 'name_in_cat' in intake.gui.cat_list.value
```

Comment:
How are the names generated for the GUI exactly - they are the names of the *sources*, I suppose? This has some merit, in that when you select a catalog in the list of entries, you expect something with that name to pop up in the cat list; still, `list(cat)` will now not necessarily align with the contents of the source list.

(I forgot to ask you in the GUI PR whether you can select catalog entries in the source list and expect them to load; also, cats were visually marked in the jupyter-widgets version as being special, different from other entries)

Reply:
I think this should be discussed in the GUI PR. I haven't thought very much about nested catalogs.